superset/charts/post_processing.py

```diff
@@ -325,9 +325,16 @@ def apply_post_process(
 
     for query in result["queries"]:
         if query["result_format"] == ChartDataResultFormat.JSON:
-            df = pd.DataFrame.from_dict(query["data"])
+            try:
+                df = pd.DataFrame.from_dict(query["data"])
+            except ValueError:  # no data error
+                return result
```

Comment:
I think it would be better to `continue` here and in line 336, since other queries might have data (and if they also don't we'll end up returning `result` unmodified).

Reply:
What is `query["data"]` in this case? Would it be preferable to first check whether `query["data"]` is valid?

Reply:
@betodealmeida if the data is None or '', is there any value in continuing this process rather than returning early? AFAICT we'll continue to get more errors down below as well.  Per @john-bodley's point, I can do a nullish check instead of the try/except if we want to be more specific to these errors.

Reply:
If you `continue` here it would skip to the next step of the `for query in result["queries"]` loop, so it wouldn't get more errors. There could be other queries in `result["queries"]` that have non-blank data.

Reply:
based on @john-bodley's comment, i added a few more tests when data is `None` and found a case where it errors later in the code, so I put a nullish check like he suggested instead of the try/except.